docs/ecosystem-eslint-plugin-testing-library.md

```diff
@@ -0,0 +1,23 @@
+---
+id: ecosystem-eslint-plugin-testing-library
+title: eslint-plugin-testing-library
+---
+
+[`eslint-plugin-testing-library`][gh] is an ESLint plugin for Testing Library
+that helps users to follow best practices and anticipate common mistakes when
+writing tests.
+
+![eslint-plugin-testing-library example showing an error](/img/eslint-plugin-screenshot.png)
+
+This plugin includes:
+
+- several rules for linting Testing Library specific code
```

Comment:
```suggestion
- Several rules for linting Testing Library specific code.
```

same thing in the other ones 😇 sry for the nitpicking.

Reply:
No problem! done too